client/__tests__/e2e/e2e.test.ts

```diff
@@ -101,6 +104,143 @@ const testURLs = {
   "super-cool-spatial.cxg": pageURLSpatial,
 };
 
+describe(`Testing CellGuideCXG at ${pageURLCellGuide}`, () => {
```

Comment:
Maybe we should put the CellGuideCXG test in its own test file?

Reply:
Not sure how to config playwright to run the separate file - if you think it's important, perhaps you could commit it to this branch so I can learn? 👼